packages/office-ui-fabric-react/src/components/Calendar/Calendar.scss

```diff
@@ -399,6 +391,7 @@
       @include ms-u-borderBox;
       @include border-right(1px, solid, $ms-color-neutralLight);
       width: 220px;
+      min-height: 230px; // Ensure the same height for months that span 4 and 5 weeks.
```

Comment:
Just to really annoy you, I'm not realizing that this is sort of a 'magic number'. Can we make a variable that defines the height of one day, and then use that both for the day and for calculating what six rows is equal to? I wouldn't block on that, but it's something we can do to make sure this doesn't break if the days are made larger in the future. 😄